iib/workers/tasks/build_merge_index_image.py

```diff
@@ -135,7 +135,7 @@ def _add_bundles_missing_in_source(
     )
     _build_image(base_dir, 'index.Dockerfile', request_id, arch)
     _push_image(request_id, arch)
-    _create_and_push_manifest_list(request_id, [arch])
+    _create_and_push_manifest_list(request_id, [arch], [])
```

Comment:
why is here an empty list?

Reply:
because it's not needed here. Final call of _create_and_push_manifest_list is in handle_merge_request